lib/internal/bootstrap/loaders.js

```diff
@@ -312,11 +305,6 @@
             return descriptor;
           },
           get: (target, prop, receiver) => {
-            if (receiver === proxy || (this.reflect !== undefined &&
-                this.reflect.namespace !== undefined &&
-                receiver === this.reflect.namespace)) {
-              receiver = target;
-            }
             const value = ReflectGet(target, prop, receiver);
             if (prop === Symbol.toStringTag &&
```

Comment:
👆 It doesn't look like native `module.exports` error on `get/set` so the guards here can be removed.